angularjs-1.5-basics/src/app/landing-page/common/convertToHtml.filter.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var convertToHtml = function ($sce) {
+    return function (input) {
+        return $sce.trustAsHtml(input);
+    };
+};
+
+angular.module('landing-page')
```

Comment:
__[jshint]__
* 'angular' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->